app/models/recipient.rb

```diff
@@ -7,6 +7,10 @@ def notify! twilio = Message::SendNotificationTextMessage.new
     target_number = person.phone
     target_number = "+1#{person.phone}" if /\A\d{10}/.match target_number
```

Comment:
person.phone has the channel.id you're looking for.

Reply:
Thanks!